lib/engine/ability/tile_lay.rb

```diff
@@ -5,10 +5,10 @@
 module Engine
   module Ability
     class TileLay < Base
-      attr_reader :hexes, :tiles, :free, :discount, :special, :connect, :blocks, :reachable, :cost
+      attr_reader :hexes, :tiles, :free, :discount, :special, :connect, :blocks, :reachable, :must_lay_together, :cost
 
       def setup(tiles:, hexes: nil, free: false, discount: nil, special: nil,
-                connect: nil, blocks: nil, reachable: nil, cost: 0)
+                connect: nil, blocks: nil, reachable: nil, must_lay_together: false, cost: 0)
```

Comment:
should this default to true? what are the inverses?

Reply:
My preference is to default everything to off, so no surprises as others add new games. What do you think?